cmd/soroban-cli/src/commands/network/container/shared.rs

```diff
@@ -148,3 +159,11 @@ async fn check_docker_connection(docker: &Docker) -> Result<(), bollard::errors:
         }
     }
 }
+
+pub fn get_container_name(container_name_arg: Option<String>, network: Option<Network>) -> String {
+    if let Some(container_name) = container_name_arg {
+        container_name.to_string()
+    } else {
+        format!("stellar-{}", network.unwrap())
```

Comment:
This is safe to do because clap will ensure that one of the two are defined. However, I would use `expect` just so that if it does happen we'll get a message and not just a panic.